client/views/ProfileView.vue

```diff
@@ -1,5 +1,6 @@
 <script setup>
-
+import SvgIcon from '@jamescoyle/vue-icon'

```

Comment:
Revert the changes, we don't need them at ProfileView yet.

Aside this one everything LGTM